sbot/internal/board_manager.py

```diff
@@ -416,6 +416,7 @@ def find_output(self, identifier: str, idx: int) -> OutputIdentifier:
         :raises ValueError: If the output does not exist on the board.
         :raises KeyError: If no board with the given identifier is registered.
         """
+        assert idx >= 0, "Output identifiers must be positive"
```

Comment:
Suggestion: Use a proper exception
```suggestion
        if idx < 0:
            raise ValueError("Output identifiers must be positive")
```